docs/source/traces.rst

```diff
@@ -146,6 +146,24 @@ Get
 
     $ st2 trace get <trace-id>
 
+* View the causation chain in a trace for an action execution. Similarly for rule and trigger-instance.
```

Comment:
Yeah I need to work on the verbiage for sure. I don't have a succinct way of expressing this idea. I will take a stab at this once I get to using RuleEnforcement. Will likely have given some thought on how to express this better anyway by then.